packages/strapi-plugin-graphql/services/Query.js

```diff
@@ -178,13 +195,15 @@ module.exports = {
 
       // Plural.
       return async (ctx, next) => {
-        ctx.params = this.amountLimiting(ctx.params);
-        ctx.query = Object.assign(
-          this.convertToParams(_.omit(ctx.params, 'where')),
-          ctx.params.where,
+        const queryOpts = {};
+        queryOpts.params = this.amountLimiting(ctx.params);
+        queryOpts.query = Object.assign(
+          {},
+          this.convertToParams(_.omit(queryOpts.params, 'where')),
+          this.convertToQuery(queryOpts.params.where)
         );
 
-        return controller(ctx, next);
+        return controller(Object.assign({}, ctx, queryOpts, { send: ctx.send }), next);
```

Comment:
Add a param to indicate that we don't want to populate all the relations when we're on GraphQL context.